main.py

```
@@ -6,6 +6,7 @@
 sys.path.append(str(module_path))
 
 from fastapi import FastAPI
+from fastapi import HTTPException
```

Ruff (0.11.9)

Module level import not at top of file

(E402)